src/content/guides/output-management.md

```diff
@@ -77,7 +77,7 @@ __dist/index.html__
 +     <script src="./print.bundle.js"></script>
     </head>
     <body>
--     <script src="./bundle.js"></script>
+-     <script src="./main.js"></script>
```

Comment:
This was correct, it extends asset-management guide and it says so at the top of the guide